tests/suites/test_suite_ecp.function

```diff
@@ -70,6 +70,9 @@ void ecp_invalid_param( )
     size_t olen;
     unsigned char buf[42] = { 0 };
 
+    mbedtls_ecp_group_init( &grp );
```

Comment:
Are the initialized structures still truly invalid? The tests pass but that's because `invalid_fmt` and `invalid_side` (above) are checked first in the respective functions that are tested. The tests are named "ECDH - Invalid parameters" and "ECP invalid params" respectively which implies that all of the parameters should be invalid.

Would it be better to `memset()` the structures to `0` so that they have definitely-invalid `NULL` pointers?

Reply:
I agree with David that `memset()` would be better, but also, these tests aren't very thorough (as above).

You know what they say about cans of worms, or about pulling on threads :)